extensions/src/main/java/org/pentaho/platform/web/http/ConfigurationAdminNonOsgiProxy.java

```diff
@@ -61,7 +60,7 @@ public ConfigurationAdminNonOsgiProxy() {
       Method getProperties = configuration.getClass().getMethod( "getProperties" );
 
       return (Dictionary<String, Object>) getProperties.invoke( configuration );
-    } catch ( NoSuchMethodException | InvocationTargetException | IllegalAccessException e ) {
+    } catch ( Exception e ) {
```

Comment:
Why are we _"generifying"_ the exceptions caught ?

Reply:
When the OSGi _"world"_ isn't present (basically when `osgiListener`/`org.pentaho.platform.osgi.OSGIBoot` is removed from `systemListeners.xml`), a `RuntimeException` is thrown (I really can't recall at this moment if it is a generic `RuntimeException` or a specific one). I decided to make the catch clause the most comprehensive, because it seems to me that was the intention of the original author of the code, but I can try to be more specific.

Reply:
The way the code is set nowadays, this `getProperties()` returns `null` when faced with one of those 3 exceptions. Any others are propagated. 

We should at least check to see if on all places that reference this method, there happens to be some custom exception catching + handling  taking place.

By changing this to catch all exceptions, we may effectively be changing an existing behaviour ( mind you, I'm not saying the way the behaviour stands nowadays is "right" or "wrong", just stating a fact ). 

If so, then those places may need changing as well, as they would never get exceptions moving forward.

Reply:
We're confident: the code was done recently by @pentaho/millenniumfalcon and is only used by `PentahoWebContextFilter`, where it expects no exceptions.